### PR TITLE
#364 random tests

### PR DESCRIPTION
**Pull request by adrianopol**
#364 added some tests for `secure_random()`

### CODE REVIEW

File: src/tests/roc_core/test_random.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2015 Roc Streaming authors
```
Comment (gavv) [inline src/tests/roc_core/test_random.cpp:2]: 2015 -> 2022

File: src/tests/roc_core/test_random.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+};
+
+TEST(random, min_min) {
+    uint32_t lo_hi = 0;
+    uint32_t res;
+    bool ok = secure_random(lo_hi, lo_hi, res);
+
+    CHECK(ok);
+    LONGS_EQUAL(res, lo_hi);
+}
```
Comment (gavv) [inline src/tests/roc_core/test_random.cpp:27]: Could you please add the same check for `fast_random` to each test?

**Review by gavv — COMMENTED** [review]
Thanks! See comments below

**Comment by gavv** [comment]
CI failed:

* code is not formatted
* build failed for one of arm toolchains: https://github.com/roc-streaming/roc-toolkit/actions/runs/3610087937/jobs/6083675329

If you follow the link, you'll see this command at the beginning of the file:

```
scripts/ci_checks/docker.sh rocstreaming/toolchain-arm-bcm2708hardfp-linux-gnueabi:gcc-4.7 scripts/ci_checks/linux-arm/arm-bcm2708hardfp-linux-gnueabi-gcc-4.7.sh
```

You can run this command locally to reproduce that CI step on your machine. See here: https://github.com/roc-streaming/roc-toolkit/blob/develop/docs/sphinx/development/continuous_integration.rst

File: src/tests/roc_core/test_random.cpp (hunk 1 above)
Comment (adrianopol) [reply src/tests/roc_core/test_random.cpp]: done

File: src/tests/roc_core/test_random.cpp (hunk 2 above)
Comment (adrianopol) [reply src/tests/roc_core/test_random.cpp]: done

**Pull request merged**